### PR TITLE
Fix On key handling

### PR DESCRIPTION
**Pull request by np511**
Fixes KnightOS/KnightOS#325

### CODE REVIEW

File: src/00/keyboard.asm [hunk 1]
```diff
@@ -121,6 +124,12 @@ getKey:
     xor a
     ret
 _:
+    in a, (4)
+    bit 3, a
+    jr nz, _
+    ld a, 0
```
Comment (SirCmpwn) [inline src/00/keyboard.asm]: can be `xor a`

Comment (np511) [inline src/00/keyboard.asm]: I'll do that - saves 3 clock cycles and a byte! Yay!

**Comment by SirCmpwn** [comment]
?

**Comment by np511** [comment]
I'm not very good with git. I was trying to pull the updates from here into my fork and I have no idea what I did.

**Comment by np511** [comment]
I have no problems with git unless I'm working on a fork. Which I am.

**Comment by SirCmpwn** [comment]
https://github.com/SirCmpwn/sway/blob/master/CONTRIBUTING.md#pull-requests

**Pull request closed without merging**